daq/test_modules/ipaddr_module.py

```diff
@@ -85,6 +86,15 @@ def _dhcp_port_toggle_test(self):
         self.host.connect_port(True)
         self._ip_callback = self._next_test
 
+    def _dhcp_long_wait_test(self):
+        self._set_timeout()
+        if not self.host.connect_port(False):
+            self._logger('disconnect port not enabled')
+            return
+        time.sleep(self._get_lease_time())
```

Comment:
This wasn’t part of the 3 extended DHCP test when I was implementing these. I thought we agreed on this test was fundamentally no different from the port_toggle considering  when you disconnect and reconnect the port, regardless of how much wait time, the device is expected to request DHCP immediately. 
time.sleep is also a blocking call so waiting for an extended period of time in python shouldn’t be done this way.

Reply:
@spectre-storm @pisuke Could you please comment on this? And if this is the case, how to go about implementing the dhcp_long test?